public/data/navigation.json

```diff
@@ -111,4 +111,4 @@
       }
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
**Example** In case of necessary changes, comments will be added in this manner.